packages/web-app-files/src/components/SideBar/Audio/AudioMetaPanel.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="files-sidebar-panel-audio">
+    <dl class="audio-data-list">
+      <template v-if="title">
+        <dt v-text="$gettext('Title')" />
+        <dd v-text="title" />
+      </template>
+      <template v-if="duration">
+        <dt v-text="$gettext('Duration')" />
+        <dd v-text="duration" />
+      </template>
+      <template v-if="artist">
+        <dt v-text="$gettext('Authors')" />
+        <dd v-text="artist" />
+      </template>
+      <template v-if="album">
+        <dt v-text="$gettext('Album')" />
+        <dd v-text="album" />
+      </template>
+      <template v-if="genre">
+        <dt v-text="$gettext('Genre')" />
+        <dd v-text="genre" />
+      </template>
+      <template v-if="year">
+        <dt v-text="$gettext('Year recorded')" />
+        <dd v-text="year" />
+      </template>
+      <template v-if="track">
+        <dt v-text="$gettext('Track')" />
+        <dd v-text="track" />
+      </template>
+      <template v-if="disc">
+        <dt v-text="$gettext('Disc')" />
+        <dd v-text="disc" />
+      </template>
+    </dl>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, inject, Ref, unref } from 'vue'
+import { Resource } from '@ownclouders/web-client'
+import { Duration } from 'luxon'
+
+export default defineComponent({
+  name: 'AudioMetaPanel',
+  setup() {
+    const resource = inject<Ref<Resource>>('resource')
+
+    const album = computed(() => {
+      return unref(resource).audio.album
+    })
+
+    const artist = computed(() => {
+      return unref(resource).audio.artist
+    })
+
+    const albumArtist = computed(() => {
+      return unref(resource).audio.albumArtist
+    })
+
+    const genre = computed(() => {
+      return unref(resource).audio.genre
+    })
+
+    const title = computed(() => {
+      return unref(resource).audio.title
+    })
+
+    const duration = computed(() => {
+      const milliseconds = unref(resource).audio.duration
+      if (milliseconds) {
```

Comment:
I am more a fan of early returns, maybe we can swap the condition and return '' early